package.json

```diff
@@ -22,9 +22,11 @@
   ],
   "dependencies": {
     "csv-parse": "^4.8.8",
-    "discord.js": "^11.6.3",
-    "luxon": "^1.22.0",
-    "node-fetch": "^2.6.0"
+    "discord.js": "^11.6.4",
+    "eslint": "^6.8.0",
```

Comment:
Should remain a peerDependency since it isn't required (and because it has a billion deps that we don't need to install on Jack's server)